src/main/java/io/opentracing/contrib/spanmanager/tracer/ManagedSpanBuilder.java

```diff
@@ -79,9 +81,6 @@ public SpanBuilder asChildOf(Span parent) {
     }
 
     public SpanBuilder addReference(String referenceType, SpanContext context) {
```

Comment:
Thanks @pavolloffay good point. Probably started out as Java-5 compatible and forgot to add interface `@Override` later.